src/mobility/utils.ts

```diff
@@ -70,7 +70,7 @@ export const getAvailableVehicles = (
     .reduce((sum, count) => sum + count, 0) ?? 0;
 
 export const getRentalAppUri = <T extends {rentalUris?: RentalUrisFragment}>(
-  t: T | undefined,
+  t: T | undefined | null,
```

Comment:
In general we use `undefined` throughout the app, not `null`. If there is some logic depending on whether the return value is `undefined` or `null` it should preferably be modelled in a return type instead.